boards/arm/stm32/nucleo-f446re/Kconfig

```diff
@@ -31,4 +31,26 @@ config NUCLEO_F446RE_AJOY_MINBUTTONS
 		minimal set:  SELECT (joystick down), FIRE (BUTTON B), and JUMP
 		(BUTTON A).
 
+config STM32_ROMFS
+	bool "Automount baked-in ROMFS image"
+	default n
+	depends on FS_ROMFS
+	---help---
+		Select STM32_ROMFS_IMAGEFILE, STM32_ROMFS_DEV_MINOR, STM32_ROMFS_MOUNTPOINT
+
+config STM32_ROMFS_DEV_MINOR
+	int "Minor for the block device backing the data"
+	depends on STM32_ROMFS
+	default 64
+
+config STM32_ROMFS_MOUNTPOINT
+	string "Mountpoint of the custom romfs image"
+	depends on STM32_ROMFS
+	default "/rom"
+
+config STM32_ROMFS_IMAGEFILE
+	string "ROMFS image file to include into build"
+	depends on STM32_ROMFS
+	default "../../../../../rom.img"
```

Comment:
seems like default is pointing somewhere out of tree

Reply:
fixed